updater/lib/dependabot/service.rb

```diff
@@ -2,6 +2,7 @@
 
 require "terminal-table"
 require "dependabot/api_client"
+require "active_support/core_ext"
```

Comment:
This is necessary to import `delegate`... for some reason it was working fine without an explicit import under `v6`, but under `v7` requires an explicit import.